panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -446,13 +460,27 @@
                         if (expWebpart.length === 1) {
                             let wp = new LABKEY.QueryWebPart({
                                 renderTo: 'webpart_'+ expWebpart[0].webPartId,
-                                title: "The searched peptide " + searchCriteriaString + " appeared in the following experiments",
+                                title: 'Panorama Public Experiments',
                                 schemaName: 'panoramapublic',
                                 queryName: 'peptideSearch',
-                                showFilterDescription: false,
+                                showFilterDescription: true,
                                 containerFilter: LABKEY.Query.containerFilter.currentAndSubfolders,
                                 parameters: peptideParameters,
                                 success: function () {
+                                    //Make the filter description user-friendly (ex: from "?peptideSequence = rrr" to using "Peptide Sequence = rrr")
+                                    let els = document.getElementsByClassName('lk-region-context-action');
```

Comment:
Let's add test coverage that checks the description too. This approach seems potentially fragile so good to validate that we keep finding the expect elements with those CSS classes

Reply:
Yup, Sweta is going to a sanity check on the updates and add the test coverage for this